HealthMate/HealthMate.API/Validators/FoodItemViewModelValidator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using FluentValidation;
+using HealthMate.API.ViewModels.FoodItem;
+
+namespace HealthMate.API.Validators
+{
+    public class FoodItemViewModelValidator : AbstractValidator<FoodItemViewModel>
+    {
+        public FoodItemViewModelValidator()
+        {
+            RuleFor(f => f.Name)
+                .NotEmpty()
+                .NotNull()
+                .Matches("^[a-zA-Z ]*$")
+                .WithMessage("Food name can only contain letters and spaces.")
+                .MinimumLength(3)
+                .MaximumLength(100)
+                .Must(StartWithUppercase)
+                .WithMessage("Food name must start with an uppercase letter.");
```

Comment:
It's better to store such messages in resource files, or, at least, in constants.